packages/backend/src/testutils.js

```diff
@@ -70,6 +78,24 @@ export const createRecipe = (userId, folder, fromUserId) => {
   });
 };
 
+export const createShoppingList = (userId) => {
+  return ShoppingList.create({
+    userId,
+    title: randomString(20),
+  }).then(function (shoppingList) {
+    return shoppingList;
+  });
+};
+
+export const createShoppingCollaborator = (list) => {
+  return ShoppingList_Collaborator.create({
+    shoppingListId: list.id,
+    userId: list.userId,
```

Comment:
The collaborator for a shopping list would always be a different user than the user who owns the list. I think having this function take `shoppingListId` and `userId` as arguments would be better here.

Reply:
done!
thank you for your review!
not sure why the build and test step on CI not running for me though

Reply:
I managed to run a circleCI step and it's all passing, but it does not report back to GitHub and I would never be able to merge this way
is it how you are running these checks and was its problem you encountered?

Reply:
CircleCI wasn't enabled for PRs from community members. I've gone ahead and enabled CircleCI for community pull requests, so your next push should run!

Reply:
I went ahead and merged master into this branch to trigger a workflow run.

Reply:
supercool it passes! thank you :)